cmd/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+func main() {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Println("Golang Bootcamp Shell")
+	fmt.Println("---------------------")
+
+	for {
+		fmt.Print("-> ")
+		input, _ := reader.ReadString('\n')
+		// convert CRLF to LF
+		input = strings.Replace(input, "\n", "", -1)
+
+		if input != "" {
+			result, e := handle(input)
+			if e == nil {
+				fmt.Println(result)
+			} else {
+				fmt.Println(e.Error())
+			}
+		}
+
+	}
+
+}
+
+func handle(input string) (result string, e error) {
+
+	r := regexp.MustCompile("\\s+")
+	arg := r.Split(input, -1)
+
+	if len(arg) == 3 {
+		a, error := strconv.Atoi(arg[0])
+		if error != nil {
+			e = errors.New("sorry, please type valid operand 1")
+		}
+
+		b, error := strconv.Atoi(arg[2])
+		if error != nil {
+			e = errors.New("sorry, please type valid operand 2")
+		}
```

Comment:
parsing for arg[0] and arg[2] code are the same, better keep it DRY by separate to a function